### ISSUE TITLE
Multipart request Issue 

### ISSUE THREAD

**Issue by zecloud** [Azure/azure-functions-python-worker#437]
#### Investigative information

Please provide the following:

- Timestamp:
- Function App name:
- Function name(s) (as appropriate): Http Trigger
- Core Tools version:  2.7.1158

#### Repro steps

Provide the steps required to reproduce the problem:

Create an Http trigger with the code below

Send a multipart form with a file 
I tried with curl
curl -F filepond=@monalisanight.jpg http://localhost:7071/api/fpupload
and with filepond component in react 
https://github.com/pqina/filepond

#### Expected behavior

It should save the same file in a blob and in the local drive

#### Actual behavior

The file is saved with a wrong size  like approximately double size of the original size

#### Related information 

Provide any related information 

* Links to source
* Contents of the requirements.txt file
* Bindings used

<details>
<summary>Source</summary>

```
 "scriptFile": "__init__.py",
  "bindings": [
    {
      "authLevel": "function",
      "type": "httpTrigger",
      "direction": "in",
      "name": "req",
      "methods": [
        "post"
      ]
    },
    {
      "name": "outputblob",
      "type": "blob",
      "path": "test/toto.jpg",
      "connection": "AzureWebJobsStorage",
      "direction": "out",
      "dataType": "binary"
    },
    {
      "type": "http",
      "direction": "out",
      "name": "$return"
    }
```python
def main(req: func.HttpRequest,outputblob: func.Out[func.InputStream]) -> func.HttpResponse:
    logging.info('Python HTTP trigger function processed a request.')
    filetoup = req.files['filepond']
    filetoup.save("/mnt/c/experiments/toto.jpg")
    filetoup.stream._file.seek(0)
    dataout=filetoup.stream._file.read()
    outputblob.set(dataout)
    return func.HttpResponse(
               status_code=200

    )
```

```txt
azure-functions==1.0.0a4
azure-functions-worker==1.0.0a4
grpcio==1.14.2
grpcio-tools==1.14.2
protobuf==3.6.1
six==1.11.0
```

</details>

**Comment by maiqbal11** [comment]
We have a known issue where body is not returned correctly for `image/png` and `application/octet-stream`: https://github.com/Azure/azure-functions-python-worker/issues/380

**Comment by asavaritayal** [comment]
> We have a known issue where body is not returned correctly for image/png and application/octet-stream: #380

@maiqbal would you suggest that this is a duplicate or worth revisiting after #380 is resolved?

**Comment by maiqbal11** [comment]
Believe this should be a duplicate. @zecloud, what is your content type?

**Comment by zecloud** [comment]
content-type is multipart/form-data and i tried with jpg and png 
i found this old issue that seems to be a similar problem in nodejs but closed maybe it's still related https://github.com/azure/azure-functions-host/issues/2009

**Comment by maiqbal11** [comment]
The handling for multipart/form-data will be the same as application/octet-stream in this case. Will need to add a fix for this in the host.

**Comment by maiqbal11** [comment]
Closing a duplicate of https://github.com/Azure/azure-functions-python-worker/issues/380. Fix should be available as part of the next release.

**Comment by daker** [comment]
@maiqbal11 can you please tell when the next release will be? i am facing the same issue using :

Azure Functions Core Tools (2.7.1373 Commit hash: cd9bfca26f9c7fe06ce245f5bf69bc6486a685dd)
Function Runtime Version: 2.0.12507.0

**Comment by maiqbal11** [comment]
@daker, I believe should be able to update it by end-of-day today. The feeds are being updated with the latest release: https://github.com/Azure/azure-functions-core-tools/releases.

**Comment by daker** [comment]
@maiqbal11 yes it works as expected!

**Comment by daker** [comment]
@maiqbal11 do you know when it's going live on azure? i think i am still having the same issue when i publish the function on azure

**Comment by maiqbal11** [comment]
@daker, we are rolling out this week. I believe it should be out in all regions by next week unless we hit some snags.
